hs-bindgen/fixtures/attributes/attributes/Example.hs

```diff
@@ -157,7 +157,9 @@ instance ( TyEq ty ((HsBindgen.Runtime.HasCField.CFieldType Bar) "bar_i")
   getField =
     HsBindgen.Runtime.HasCField.ptrToCField (Data.Proxy.Proxy @"bar_i")
 
-{-| __defined at:__ @attributes\/attributes.h:22:9@
+{-| __C declaration:__ @baz@
+
+    __defined at:__ @attributes\/attributes.h:22:9@
```

Comment:
This name was initially omitted from the documentation because the declaration was anonymous. However, we now refer to the enclosing typedef instead, which I think is fine (better, in fact).